u_0 [vhme] error in (finalize_query, query_tel): KELNotFound(SelfAddressing("EHfGKYdm_fMvINYszsOpVdWHh2PnIKvSe9Wpug7XSbRf")). Request was: EventSeal { prefix: SelfAddressing("EHfGKYdm_fMvINYszsOpVdWHh2PnIKvSe9Wpug7XSbRf"), sn: 1, event_digest: "EFzmaxOKFRP_EJhZv4IQ5I0FKxN9ieYdEjnJZKfUFHfp" }
Attempt 1 failed: SendingError(ActorInternalError(GeneralError("Either Watcher Error or NoUpdates"))). Retrying in 938 ms...
